test/geo_shape_test.rb

```diff
@@ -0,0 +1,145 @@
+require "pp"
+require_relative "test_helper"
+
+class GeoShapeTest < Minitest::Test
+
+  def test_geo_shape
+    regions = [
+      {name: "Region A", text: "The witch had a cat", territory: "30,40,35,45,40,40,40,30,30,30,30,40"},
+      {name: "Region B", text: "and a very tall hat", territory: "50,60,55,65,60,60,60,50,50,50,50,60"},
+      {name: "Region C", text: "and long ginger hair which she wore in a plait.",  territory: "10,20,15,25,20,20,20,10,10,10,10,20"},
+    ]
+    store regions, Region
```

Comment:
It'd be nice to put region creation in a `setup` method, and split the assertions into separate tests - `test_circle`, `test_envelope`, etc.